tools/nuget/validate_package.py

```diff
@@ -204,6 +224,7 @@ def validate_tarball(args):
         is_windows_ai_package,
         is_gpu_package,
         is_dml_package,
+        False,
```

Comment:
nit: using a variable, e.g., `is_training_package = False` (as is done for `is_dml_package`) would be clearer than literal `False`